config/canonical.json

```
@@ -31356,7 +31358,7 @@
     "tags": {
       "brand": "Tesco",
       "name": "Tesco",
-      "shop": "supermarket"
+      "shop": "supermarket",
```

no trailing `,` here
```suggestion
      "shop": "supermarket"
```

---

:thinking: That's odd, this is the trailing comma from master fixed in #1914, how was it committed here?

---

Yeah I was wondering that too...
The merges are coming fast and furious today, so I'm not surprised that there is a little bit of breakage and that every time we `npm run build` things get a little shuffled around.